views/index.ejs

```diff
@@ -531,6 +535,7 @@
     const _host = "<%= host %>";
     const _fabricApiUrl = "<%= fabricApiUrl %>";
     const _firebaseConfig = <%- firebaseConfig %>;
+    const _oauth_config = <%- oauthConfig %>;
```

Comment:
We are not using it anywhere on the client side, are we? 

The same goes for `_token`?

Reply:
We need to authConfig to initialize the oicd instance.

Reply:
Oh yeah, I saw the `_oauth_config` usage. We need to remove only the `_token`.